src/euroeval/benchmarker.py

```diff
@@ -366,31 +366,37 @@ def benchmark(
             dataset_names=benchmark_config.datasets
         )
 
+        total_benchmarks = len(model_ids) * len(dataset_configs)
+        current_benchmarks = 0
```

Comment:
Can we rename this to something like `num_finished_benchmarks`? The `current_benchmarks`  doesn't really convey the meaning properly, I think.